airbyte-integrations/connectors/source-gridly/source_gridly/spec.yaml

```diff
@@ -0,0 +1,15 @@
+documentationUrl: https://docsurl.com
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Gridly Spec
+  type: object
+  required:
+    - api_key
+    - view_id
+  properties:
+    api_key:
+      type: string
+      description: API Key
+    view_id:
+      type: string
+      description: View ID
```

Comment:
I'm looking the API for Gridly and there are ways to retrieve all views and use a substream to read individual views without informing in the config.
Also there are a lot of other endpoints you can implement for this connector:
* database
* grid
* ...

Let me know if you need any assistance to implement the review I requested.

Reply:
I updated the config, users now can specify a grid id and they can select which views to sync records

Reply:
Ok Thanks!